app/controllers/admin/bulk_line_items_controller.rb

```diff
@@ -33,7 +33,9 @@ def update
       # and https://www.postgresql.org/docs/current/static/sql-select.html#SQL-FOR-UPDATE-SHARE
       order.with_lock do
         if @line_item.update(line_item_params)
-          order.update_distribution_charge!
+          order.update_line_item_fees! @line_item
+          order.update_order_fees!
+          order.update!
```

Comment:
In this case we're updating a single line item in a completed order. We only need to update the fees on that _one_ line item and then update the order-level fees, instead of deleting and re-creating all fees from scratch.